hawkbit-repository/src/test/java/org/eclipse/hawkbit/repository/utils/MultipleInvokeHelper.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2011-2016 Bosch Software Innovations GmbH, Germany. All rights reserved.
+ */
+package org.eclipse.hawkbit.repository.utils;
+
+import java.util.concurrent.Callable;
+
+/**
+ * Helper to call a request multiple times regarding a given condition until
+ * timeout is reached.
+ * 
+ * @author Jonathan Knoblauch
```

Comment:
[WARNING] Missing header in: /home/ubuntu/hawkbit/hawkbit-repository/src/test/java/org/eclipse/hawkbit/repository/utils/MultipleInvokeHelper.java
[WARNING] Missing header in: /home/ubuntu/hawkbit/hawkbit-repository/src/test/java/org/eclipse/hawkbit/repository/utils/SuccessCondition.java